### DISCUSSION TITLE
visualizer compiler warning over missing file: ~progguide::exampledata::accounts (cosmetic?)

### DISCUSSION THREAD

**Discussion by eliam** [hpcc-systems/Visualizer#16] [Q&A] [answered]
When using the visualizer, I get the following compiler warning: "Warning:  Error looking up file progguide::exampledata::accounts in DFS - file not found (36, 28), 3142, ".

Using Docker image created from hpcc-base:7-xenial + hpccsystems-platform-community_7.12.24-1bionic_amd64.deb

Visualizer installed using: ecl bundle install https://github.com/hpcc-systems/Visualizer.git

Visualizer appears to be working so I'm assuming this is cosmetic.

**Comment by GordonSmith** [comment, marked as answer]
Correct - one of the self tests is assuming that the "GenData" from the Programmers Guide has been executed within the environment and it can be safely ignored.
